src/main/scala/mesosphere/marathon/state/AppDefinition.scala

```diff
@@ -274,6 +275,8 @@ case class AppDefinition(
 
   private def portIndices: Range = container.flatMap(_.hostPorts).getOrElse(portNumbers).indices
 
+  def hostPorts: Seq[Option[Int]] = container.flatMap(_.hostPorts).getOrElse(portNumbers.map(Some(_)))
```

Comment:
there's probably other places this is useful too, so I preserved the arity instead of flattening